bin/templates/cordova/lib/build.js

```diff
@@ -530,8 +609,54 @@ module.exports.findBestApkForArchitecture = function(buildResults, arch) {
     throw new Error('Could not find apk architecture: ' + arch + ' build-type: ' + buildResults.buildType);
 };
 
+function PackageInfo(keystore, alias, keystorePassword, password, storeType) {
+    keystore = keystore.replace(/\\/g, '\\\\');
+    this.keystore = {
+        'name': 'key.store',
+        'value': keystore
+    };
+    this.alias = {
+        'name': 'key.alias',
+        'value': alias
+    };
+    if (keystorePassword) {
+        this.keystorePassword = {
+            'name': 'key.store.password',
+            'value': keystorePassword
+        };
+    }
+    if (password) {
+        this.password = {
+            'name': 'key.alias.password',
+            'value': password
+        };
+    }
+    if (storeType) {
+        this.storeType = {
+            'name': 'key.store.type',
+            'value': storeType
+        };
+    }
+}
+
+PackageInfo.prototype = {
+    toProperties: function() {
+        var self = this;
+        var result = '';
+        Object.keys(self).forEach(function(key) {
+            if (self[key]) {
```

Comment:
Probably should not require the value to be non-empty. Perfectly valid to have `key.alias.password=` in the file (although not recommended :P)